source/usage-examples/findOneAndUpdate.txt

```diff
@@ -0,0 +1,42 @@
+============================
+Update a Document Atomically
+============================
+
+.. default-domain:: mongodb
+
+A single document can be updated and returned in one single atomic
```

Comment:
Can you suggest an alternative sentence? 
In my opinion, the suggested sentence ("A single document can be updated and returned in an atomic operation using the findOneAndUpdate() method.") goes against our anti-jargon philosophy of these usage examples because it uses a complex term (atomic) without providing an explanation as to what it is.